app/models.py

```diff
@@ -2991,5 +3002,34 @@ def extend_subscription(
             },
         )
 
+    @classmethod
+    def downloaded_provider_complaint(cls, admin_user_id: int, complaint_id: int):
+        cls.create(
+            admin_user_id=admin_user_id,
+            action=AuditLogActionEnum.download_provider_complaint.value,
+            model="ProviderComplaint",
+            model_id=complaint_id,
+            data={},
+        )
 
-# endregion
+
+class ProviderComplaintState(EnumE):
+    new = 0
+    reviewed = 1
+
+
+class ProviderComplaint(Base, ModelMixin):
+    __tablename__ = "provider_complaint"
+
+    user_id = sa.Column(sa.ForeignKey("users.id"), nullable=False)
+    state = sa.Column(sa.Integer, nullable=False)
```

Comment:
should we use `ProviderComplaintState` here for the state column?

Reply:
We're using `ProviderComplaintState` for the state. Like in `provider_complaint.py:230`

Reply:
We can also enforce this in the data model, like in https://github.com/simple-login/app/blob/eab7606f93b1dad21522ad319e2a5c2eff2bdce5/app/models.py#L464

This isn't the case everywhere though so this stays totally optional.

Reply:
This is only setting the default to the enum value isn't it? Nothing prevents it from doing `state=3` somewhere in the code afaik.

Reply:
I've added the default.